### PR TITLE
Convert Inline Table Rows to Individual CodeLines

### PR DESCRIPTION
**Pull request by chidozieononiwu**
- Split Swagger Inline Tables into separated code lines.

- Add Background task that
   - Updates Precomputed Line Numbers Metadata.
   - Updates referencing comments
   - Updates Precomputed Diff Line Metadata.
   
   Resolves https://github.com/Azure/azure-sdk-tools/issues/5831
   
   Demo Link https://apiviewuat.azurewebsites.net/Assemblies/Review/1872ee16e46e4b288026e05aff3a5776?revisionId=14688791636e4b73ae1e9426155031f7&diffRevisionId=9a716effb6174b98ae0da797cf5903d3&doc=False&diffOnly=False

BEFORE
SwaggerTablesBefore

AFTER
SwaggerAfter
SwaggerAfterTwo

### CODE REVIEW

**Comment by benbp** [comment]
This is pretty cool!

**Comment by mikekistler** [comment]
I tried out the demo and it looks great!

File: src/dotnet/APIView/APIViewWeb/HostedServices/SwaggerReviewsBackgroundHostedService.cs [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        {
+            if (!_isDisabled)
+            {
+                try 
+                {
+                    await _reviewManager.UpdateSwaggerReviewsMetaData();
+                }
+                catch (Exception ex)
+                {
+                    _telemetryClient.TrackException(ex);
```
Comment (benbp) [inline src/dotnet/APIView/APIViewWeb/HostedServices/SwaggerReviewsBackgroundHostedService.cs:39]: Do we track success as well under the hood so we can compare percentages?

File: src/dotnet/APIView/APIViewWeb/Managers/ReviewManager.cs [hunk 2]
```diff
@@ -385,6 +389,85 @@ public async Task UpdateReviewBackground(HashSet<string> updateDisabledLanguages
...
+                                    if (rowCount > 1)
+                                    {
+                                        comment.ElementId = comment.ElementId + $"-tr-{rowCount - 1}";
+                                        await _commentsRepository.UpsertCommentAsync(comment);
+                                        break;
+                                    }
+                                }
+                            }
+                            if (rowCount > 0)
+                                break;
```
Comment (benbp) [inline src/dotnet/APIView/APIViewWeb/Managers/ReviewManager.cs:461]:
```suggestion
                            if (rowCount > 0)
                            {
                                break;
                            }
```

**Review by benbp — APPROVED** [review]

File: src/dotnet/APIView/APIViewWeb/Managers/ReviewManager.cs [hunk 3]
```diff
@@ -385,6 +389,85 @@ public async Task UpdateReviewBackground(HashSet<string> updateDisabledLanguages
             }            
         }
 
+        public async Task UpdateSwaggerReviewsMetaData()
+        {
+            IList<ReviewModel> reviews = (await _reviewsRepository.GetReviewsAsync(isClosed: false, language: "Swagger", fetchAllPages: true)).ToList();
+            reviews.AddRange(await _reviewsRepository.GetReviewsAsync(isClosed: false, language: "Swagger", fetchAllPages: true));
```
Comment (praveenkuttappan) [inline src/dotnet/APIView/APIViewWeb/Managers/ReviewManager.cs:395]: Why do we fetch reviews twice? Did you meant to fetch closed reviews in second get reviews?

**Review by praveenkuttappan — APPROVED** [review]

File: src/dotnet/APIView/APIViewWeb/HostedServices/SwaggerReviewsBackgroundHostedService.cs (hunk 1 above)
Comment (chidozieononiwu) [reply src/dotnet/APIView/APIViewWeb/HostedServices/SwaggerReviewsBackgroundHostedService.cs:39]: I do log the number of reviews successfully processed.

**Pull request merged**